### PR TITLE
Modified DataServlet.java to return hard-coded JSON data, then modifi…

### PR DESCRIPTION
**Pull request by ekengni**
…ed script.js to fetch that JSON data.

### CODE REVIEW

File: src/main/webapp/gallery.html [hunk 1]
```diff
@@ -7,13 +7,12 @@
     <style>
     body {
    	    margin: 0;
-        background-image: url(port_images/IMG-9896.jpg);
-        background-repeat: no-repeat;
-        background-size: cover;
-        position: center;
-        overflow-y: auto;
+        background-image: url(port_images/unsplash.jpg);
```
Comment (lazau-g) [inline src/main/webapp/gallery.html]: nitpick: Remove trailing whitespaces

File: src/main/webapp/projects_research.html [hunk 2]
```diff
@@ -60,6 +60,10 @@ <h2> Google Maps API </h2>
     <p> This is where I will experimented with a an API.
         enjoy :D </p>
     <div id="map"></div>
+    <p> Servlet button from walk through. </p>
+    <button onclick = "getServerMessages()">Servlet Demo </button>
+    <div id="quote-container"></div>
```
Comment (lazau-g) [inline src/main/webapp/projects_research.html]: Do you mean to make this div a `<ul>` or `<ol>` element?

Right now, createListElement appends `<li>` elements inside the `<div>`, which isn't really valid HTML (although most browsers will understand what you're doing and display it properly).

File: src/main/webapp/gallery.html (hunk 1 above)
Comment (ekengni) [reply src/main/webapp/gallery.html]: corrected

File: src/main/webapp/projects_research.html (hunk 2 above)
Comment (ekengni) [reply src/main/webapp/projects_research.html]: corrected

**Review by lazau-g — APPROVED** [review]
Good job!

**Pull request merged**